cypress/integration/onboarding/messaging/whitespace_does_not_affect_emoji_size_spec.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
We'll re-organize the folder structure of integration soon.  With that, could you please create new folder and rename this file into something like:
```
cypress/integration/messaging/emoji_size_spec.js 
```

Reply:
Done!